package.json

```diff
@@ -114,31 +119,34 @@
+    "build": "yarn install && LANGUAGE=en-US BABEL_ENV=production NODE_ENV=production CI=1 node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js -p --progress --colors --config build/webpack.config.js",
+    "build-locales": "node ./build/props-to-es2015.js",
+    "watch": "yarn install && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev ./node_modules/.bin/webpack --watch --progress --colors --config build/webpack.config.js",
     "test": "yarn install && NODE_ENV=test yarn run jest",
     "debug": "yarn install && NODE_ENV=test yarn run jest --watch",
     "setup-travis": "cd functional-tests && yarn install && node app.js",
-    "selenium-build": "BABEL_ENV=production NODE_ENV=production CI=1 node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js -p --progress --colors --config build/webpack.selenium.config.js",
+    "selenium-build": "LANGUAGE=en-US BABEL_ENV=production NODE_ENV=production CI=1 node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js -p --progress --colors --config build/webpack.selenium.config.js",
     "functional-tests": "node functional-tests/app.js & node ./node_modules/codeceptjs/bin/codecept.js run --steps",
     "functional-tests-ci": "yarn run clean && yarn run selenium-build && node ./node_modules/codeceptjs/bin/codecept.js run --steps",
     "lint": "NODE_ENV=dev ./node_modules/.bin/eslint src && ./node_modules/.bin/stylelint 'src/**/*.scss'",
     "precommit": "lint-staged",
     "commitmsg": "commitlint -e",
     "prepush": "./build/prepush.sh",
+    "pre-build": "yarn install && yarn run clean && yarn run build-locales",
     "flow": "flow",
     "flow-typed": "flow-typed",
     "flow-typed-update": "yarn add flow-typed && yarn run flow-typed update -i dev",
     "prettier": "prettier-eslint \"src/**/*.js\" --print-width 120 --single-quote --tab-width 4 --write",
     "ci": "yarn nsp && yarn run clean && yarn run lint && yarn install && yarn run test",
-    "clean": "rm -rf lib rm -rf functional-tests/lib && rm -rf reports/coverage",
+    "clean": "rm -rf lib functional-tests/lib reports/* i18n/json i18n/*.js styleguide",
     "github-release": "./node_modules/.bin/conventional-github-releaser",
     "changelog": "./node_modules/.bin/conventional-changelog -i CHANGELOG.md --same-file",
     "minor": "./build/release.sh -n && ./build/publish.sh",
     "major": "./build/release.sh -m && ./build/publish.sh",
     "patch": "./build/release.sh -p && ./build/publish.sh",
     "nsp": "nsp check --reporter summary",
-    "upgrade-brui": "./build/upgrade_brui.sh"
+    "upgrade-brui": "./build/upgrade_brui.sh",
+    "styleguide": "yarn run pre-build && LANGUAGE=en-US REACT=true BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config build/styleguide.config.js --mode development"
```

Comment:
This seems to be duplicate from the examples PR. Can you please clean up the patchsets?

Reply:
ah merge conflict! fixing it